src/openrct2-ui/UiContext.cpp

```diff
@@ -140,6 +140,28 @@ class UiContext final : public IUiContext
     void Tick() override
     {
         _inGameConsole.Update();
+
+        // the flickering frequency is reduced by 4, compared to the original
+        // it was done due to inability to reproduce original frequency
+        // and decision that the original one looks too fast
+        if (gCurrentRealTimeTicks % 4 == 0)
```

Comment:
This file needs to import `Game.h`  (where `gCurrentRealTimeTicks` is declared).

Reply:
I was thinking about it more and i think I'll actually move this to the map window as part of the update function as I'm pretty sure that's the only user